.github/workflows/test-examples-on-label.yml

```diff
@@ -14,4 +14,4 @@ jobs:
   test-examples-on-label:
     if: contains(github.event.pull_request.labels.*.name, 'run-examples')
     name: On label
-    uses: glass-dev/glass/.github/workflows/run-examples.yml@main
+    uses: ./.github/workflows/run-examples.yml
```

Comment:
This should have always been like this. Ran into issues where the updated workflow wasn't being run...